src/us/kbase/test/auth2/authcontroller/authjars

```diff
@@ -1,7 +1,7 @@
-kbase/auth2/kbase-auth2templates-0.4.1.zip
+kbase/auth2/kbase-auth2templates-0.4.3.zip
 
-kbase/auth2/kbase-auth2-0.4.1.jar
-kbase/auth2/kbase-auth2test-0.4.1.jar
+kbase/auth2/kbase-auth2-0.4.3.jar
+kbase/auth2/kbase-auth2test-0.4.3.jar
```

Comment:
The 0.4.3 jars will be created and added to the jars repo after this commit is merged.